BaseCollector.py

```diff
@@ -185,6 +185,10 @@ def get_target_tokens(self):
         self.target_tokens = self.do_request(url="http://" + os.environ['INVENTORY'] + "/target_tokens")
         return self.target_tokens
 
+    def get_vrops_list(self):
+        self.vrops_list = self.do_request(url="http://" + os.environ['INVENTORY'] + "/vrops_list")
+        return self.vrops_list
```

Comment:
This should be local only. You're always calling the function instead of the object itself.

```suggestion
        vrops_list = self.do_request(url="http://" + os.environ['INVENTORY'] + "/vrops_list")
        return vrops_list
```